setup_board

```diff
@@ -155,6 +155,12 @@ fi
 if [ "${FLAGS_skip_toolchain_update}" -eq "${FLAGS_TRUE}" ]; then
   UPDATE_ARGS+=" --skip_toolchain_update"
 fi
+if [[ "${BOARD}" == "arm64-usr" ]]; then
+  [[ ! -d /etc/portage/package.use ]] && sudo mkdir /etc/portage/package.use
+  sudo_clobber "/etc/portage/package.use/grub" <<EOF
+sys-boot/grub grub_platforms_arm64
+EOF
+fi
```

Comment:
@glevand Well, right now building the arm64 grub target requires the arm64 cross compiler which isn't in the SDK until setup_board is used. It would be nice to avoid changing grub use flags but we'll need to adjust how we bootstrap the SDK first.

One slightly different scheme that may feel less awkard is split the grub ebuild into two, with a special grub-arm64 ebuild that can depend on the cross compiler. Then no use flag twiddling is required. Off the top of my head I'm not sure which I favor. Perhaps go with this for now but it can be tweaked later.